llvm/docs/Contributing.rst

```diff
@@ -107,11 +107,19 @@ way, in which case you can still get the attention of potential reviewers by CC'
 them in a comment -- just @name them.
 
 If you have received no comments on your patch for a week, you can request a
-review by 'ping'ing the GitHub PR with "Ping". The common courtesy 'ping' rate
+review by 'ping'ing the GitHub PR with "Ping" in a comment. The common courtesy 'ping' rate
 is once a week. Please remember that you are asking for valuable time from
-other professional developers. Finally, if you do not have commit access,
-please let people know during the review and someone should commit it on your
-behalf once it has been accepted.
+other professional developers.
+
+After your PR is approved, ensure that:
+
+  * The PR title and description describe the final changes. These will be used
+    as the title and message of the final squashed commit. The titles or
+    messages of any commits in the PR will **not** be used.
+  * You have set a valid email address in your GitHub account, see :ref:`github-email-address`.
+
+Now you can merge your PR. If you do not have the ability to merge the PR, ask your
+reviewers to merge it on your behalf.
```

Comment:
I would add something like: "You will need to do this explicitly, as your reviewers will likely assume you have commit access unless you say otherwise."

Reply:
Well, it's not that we assume everyone has commit access, but that we don't want to push prematurely in case they do.